src/styles/blocks/variables.scss

```diff
@@ -0,0 +1,4 @@
+$hoverTransitionDuration: 300ms;
+$linkTextColorOnHover: #00acdc;
+$cardScaleOnHover: 1.2;
```

Comment:
You don't really need to create a variable for the scale value, you use it only once. But it's not a mistake, you can leave it like that